versioned_plugins.rb

```diff
@@ -429,6 +432,41 @@ def load_alias_definitions_for_target_plugins(plugin_names_by_type)
 
     aliases
   end
+
+  def fetch_stack_versions
+    stack_versions = Net::HTTP.get(URI.parse(VERSIONS_URL))
+    puts "Logstash version: #{get_logstash_version(stack_versions)}\n"
+    puts "ECS version: #{get_ecs_version(stack_versions)}\n"
+  end
+
+  def get_logstash_version(stack_versions)
+    version = stack_versions[/\:logstash_version: (.*?)\n/, 1]
+    @logstash_version = version.strip unless version.nil?
+  end
+
+  def get_ecs_version(stack_versions)
+    version = stack_versions[/\:ecs_version: (.*?)\n/, 1]
+    @ecs_version = version.strip unless version.nil?
```

Comment:
```suggestion
    @ecs_version = stack_versions[/\:ecs_version:\s+(.*?)\n/, 1]
```

Reply:
Thanks @jsvd ! Committed the suggested changes.
Time to learn more about Ruby :))